roles/openshift_master_facts/tasks/main.yml

```diff
@@ -6,7 +6,8 @@
     openshift_master_default_subdomain: "{{ osm_default_subdomain | default(None) }}"
   when: openshift_master_default_subdomain is not defined
 
-- fail:
+- name: Verify required variables are set
+  fail:
```

Comment:
Created https://github.com/openshift/openshift-ansible/pull/3364 and https://github.com/openshift/openshift-ansible/pull/3365 to separate out the changes unrelated to molecule.